frontend/src/setupProxy.js

```diff
@@ -0,0 +1,13 @@
+const createProxyMiddleware = require('http-proxy-middleware');
+
+const { BACKEND_HOST = 'localhost' } = process.env;
+
+module.exports = function (app) {
+  app.use(
+    '/oauth',
```

Comment:
The redirection after the OAuth doesn't seem to work for me. I get redirected to localhost:4000. We can clean this up later, I don't think this is a blocker.